release-notes/release-notes5104.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: TinyMCE 5.10.4
+title_nav: TinyMCE 5.10.4
+description: Release notes for TinyMCE 5.10.4
+keywords: releasenotes bugfixes
+---
+
+## Overview
+
+{{site.productname}} 5.10.4 was released for {{site.enterpriseversion}} and {{site.cloudname}} on Wednesday, May 4<sup>th</sup>, 2022. It includes {{site.productname}} 5.10.4 and additional changes to premium plugins. These release notes provide an overview of the changes for {{site.productname}} 5.10.4, including:
+
+- [Accompanying Premium Plugin changes](#accompanyingpremiumpluginchanges)
+- [General bug fixes](#generalbugfixes)
+- [Upgrading to the latest version of TinyMCE 5](#upgradingtothelatestversionoftinymce5)
+
+{{site.releasenotes_for_stable}}
+
+## Accompanying Premium Plugin changes
+
+The following premium plugin updates were released alongside {{site.productname}} 5.10.4.
+
+### Spell Checker Pro 2.5.1
+
+The {{site.productname}} 5.10.4 release includes an accompanying release of the **Spell Checker Pro** premium plugin.
+
+**Spell Checker Pro** 2.5.1 fixes the page focus incorrectly changing when spellchecking is triggered on multiple editors after scrolling.
```

Comment:
+In the previous release, **Spell Checker Pro** would change page focus incorrectly when spellchecking was triggered on multiple editors after scrolling.
+
+**Spell Checker Pro** 2.5.1, which accompanies this release, corrects this: page focus no longer changes in this circumstance.



The above is a sort-of Cause-Consequence-Fix-Result (CCFR) re-write. I still have questions nonetheless (arising out of ignorance).

What is ‘page focus’? I presume it means the page/object the end-user expects to take input suddenly stops being such. Is this a fair presumption?

What is ‘scrolling’? An end-user action (eg, using the mouse’s scroll wheel or dragging the scroll wheel)? Or something programmatic (eg an auto-scroll to the end of the loaded document)? Or either?

Reply:
`page focus` is what element is focused on the current page/document/window. `scrolling` is any form of page movement (keyboard, mouse, etc...), that triggers a `scroll` event.

Reply:
```suggestion
In the previous release, **Spell Checker Pro** would change page focus incorrectly when spellchecking was triggered on multiple editors after scrolling.

**Spell Checker Pro** 2.5.1, which accompanies this release, corrects this: page focus no longer changes in this circumstance.
```
I think this is what you were suggesting? The last part doesn't make sense to me though. Maybe it should be something like this instead?
```
**Spell Checker Pro** 2.5.1, which accompanies this release, corrects this so that the page focus no longer moves in this circumstance.
```

Reply:
```suggestion
In the previous release, **Spell Checker Pro** would change page focus incorrectly when spellchecking was triggered on multiple editors after a `scroll` event.

**Spell Checker Pro** 2.5.1, which accompanies this release, corrects this so that the page focus no longer moves in this circumstance.
```

Reply:
This is your suggested paragraphs with one small tweak:

s/scrolling/a `scroll` event./

The tweak goes for the developer-centric phrase as being both more cause-specific and more audience-specific.